src/Multiple/ExtraSmallMultipleFormatter.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace CultuurNet\CalendarSummaryV3\Multiple;
+
+use CultuurNet\CalendarSummaryV3\Translator;
+
+abstract class ExtraSmallMultipleFormatter
+{
+    /**
+     * @var string $langCode
+     */
+    protected $langCode;
+
+    /**
+     * @var boolean $hidePast
+     */
+    protected $hidePast;
+
+    /**
+     * @var Translator
+     */
+    protected $trans;
+
+    public function __construct(string $langCode, bool $hidePastDates)
+    {
+        $this->langCode = $langCode;
+        $this->hidePast = $hidePastDates;
```

Comment:
Is it possible we never use this in the child classes?

Reply:
Hmm, now I'm thinking we forgot to build in a feature for this.

Reply:
If you're asking if we have actual logic for this in other places: we do.

I'm seeing checks in [LargeMultipleHTMLFormatter](https://github.com/cultuurnet/calendar-summary-v3/blob/master/src/Multiple/LargeMultipleHTMLFormatter.php#L29) and [LargeMultiplePlainTextFormatter](https://github.com/cultuurnet/calendar-summary-v3/blob/master/src/Multiple/LargeMultiplePlainTextFormatter.php#L31), both subclasses of [LargeMultipleHTMLFormatter](https://github.com/cultuurnet/calendar-summary-v3/blob/master/src/Multiple/LargeMultipleFormatter.php)

I'm also seeing checks in [MediumMultipleHTMLFormatter](https://github.com/cultuurnet/calendar-summary-v3/blob/master/src/Multiple/MediumMultipleHTMLFormatter.php#L29) and [MediumMultiplePlainTextFormatter](https://github.com/cultuurnet/calendar-summary-v3/blob/master/src/Multiple/MediumMultiplePlainTextFormatter.php#L31), both subclasses of [MediumMultipleFormatter](https://github.com/cultuurnet/calendar-summary-v3/blob/master/src/Multiple/MediumMultipleFormatter.php)

But it's also present in [SmallMultipleFormatter](https://github.com/cultuurnet/calendar-summary-v3/blob/master/src/Multiple/SmallMultipleFormatter.php) while the subclasses [SmallMultipleHTMLFormatter](https://github.com/cultuurnet/calendar-summary-v3/blob/master/src/Multiple/SmallMultipleHTMLFormatter.php) and [SmallMultiplePlainTextFormatter](https://github.com/cultuurnet/calendar-summary-v3/blob/master/src/Multiple/SmallMultiplePlainTextFormatter.php) don't use it at all. I think it was just copy-pasted without thinking about it in this case.

If you're asking if we need this for `ExtraSmallMultipleFormatter`, my guess would be no since Large and Medium use it, but Small doesn't. But maybe best to check with @corneelwille ? Maybe it's actually a mistake in small and we need it in all formatters? If so, we'd need extra logic to actually hide the past dates in Small and ExtraSmall of course.

Reply:
After looking closer at the code, I don't believe we need it for Small and Medium formatters since they don't list all the dates separately. If it turns out we *do* need them, we can always fix it through a separate PR.

Reply:
> After looking closer at the code, I don't believe we need it for Small and Medium formatters since they don't list all the dates separately.

Hmm but Medium already has logic for it as I showed in my previous comment?

Okay for me not to use it in ExtraSmall, but then we could remove the constructor argument & property on that new base class no?

Reply:
Yes, I meant small and extra small (not medium, that one lists all dates separately)

Reply:
I'll see if I can get rid of the unnecessary arguments

Reply:
I added the cleanup to the follow-up PR: https://github.com/cultuurnet/calendar-summary-v3/pull/11/commits/ae59ee5eabb019fd2493b6c625873b73d3f956e1

Reply:
Thanks!